src/Packetery/Module/Carrier/CountryListingPage.php

```diff
@@ -201,6 +228,8 @@ public function render(): void {
 			'countryCode'                => __( 'Country code', 'packeta' ),
 			'setUp'                      => __( 'Set up', 'packeta' ),
 			'noActiveCountries'          => __( 'No active countries.', 'packeta' ),
+			'noCarriersFound'            => __( 'No results found!', 'packeta' ),
+			'carrierSearch'              => __( 'Search', 'packeta' ),
```

Comment:
mohlo by se to jmenovat třeba searchPlaceholder

Reply:
Prejmenovano.